raiden-cli/src/index.ts

```diff
@@ -121,9 +121,9 @@ function parseArguments() {
         desc: 'Anything else than "pfs" disables mediated transfers',
       },
       pathfindingServiceAddress: {
-        type: 'string',
-        default: 'auto',
-        desc: 'Force a given PFS to be used; "auto" selects cheapest registered on-chain',
+        type: 'array',
+        desc:
+          'Force given PFS URL list to be used, instead of using best registered from "ServiceRegistry" contract',
```

Comment:
This might get quite long, but I would add that they are sorted by priority and the first available gets used. Right?

Reply:
@andrevmatos please close this conversation independently when it is done. I don't see a reason to very this once done. :grimacing: